### PR TITLE
Clear inbox filter on new chat

### PR DESCRIPTION
**Pull request by akalin-keybase**

### CODE REVIEW

**Comment by akalin-keybase** [comment]
r @keybase/react-hackers ?

File: shared/chat/inbox/container.js [hunk 1]
```diff
@@ -234,12 +234,16 @@ const mapDispatchToProps = (dispatch: Dispatch, {focusFilter}) => ({
   loadInbox: () => dispatch(loadInbox()),
   onHotkey: cmd => {
     if (cmd.endsWith('+n')) {
+      dispatch(setInboxFilter(''))
```
Comment (chrisnojima) [inline shared/chat/inbox/container.js]: Likely the saga should do this right

Comment (akalin-keybase) [reply shared/chat/inbox/container.js]: Done!

**Comment by akalin-keybase** [comment]
PTAL

**Pull request merged**